english/fork-jam/choco-chalice/find-stick/find-stick.md

```diff
@@ -0,0 +1 @@
+You spy the perfect stick for roasting marshmallows (it does two at once!).  You begin to drool as you turn towards the giant marshmallow room.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.